UnityProject/Assets/Scripts/Messages/Server/TabUpdateMessage.cs

```diff
@@ -124,8 +125,8 @@ public static void SendToPeepers(GameObject provider, NetTabType type, TabAction
 					NetworkTabManager.Instance.Remove(provider, type, recipient);
 					break;
 				case TabAction.Update:
-					//fixme: duplication of NetTab.ValidatePeepers
-					//Not sending updates and closing tab for players that don't pass the validation anymore
+					// TODO: FIXME: duplication of NetTab.ValidatePeepers
```

Comment:
usually ``FIXME`` is also recognized by IDE's and shows in a higher priority colour than TODO

Reply:
I include both, in case someone is deliberately searching for one of them.